test/input-validation.test.js

```diff
@@ -67,3 +67,36 @@ test('not evaluate json-schema $schema keyword', t => {
     t.equal(res.payload, 'world')
   })
 })
+
+test('validation context in validation result', t => {
+  t.plan(2)
+  const fastify = Fastify()
+  // return validation context, so we can test it in response
+  fastify.setErrorHandler((err, request, reply) => reply.send(err.validationContext))
```

Comment:
Would you mind adding a few assertions here?

Reply:
Like? Sorry, please could you give me some precisions?

Reply:
You should also test that the error is what you are expecting :)